addons/medical_statemachine/Statemachine.hpp

```diff
@@ -101,10 +101,6 @@ class ACE_Medical_StateMachine {
             condition = QFUNC(conditionExecutionDeath);
             events[] = {QEGVAR(medical,FatalInjury)};
         };
-        class Bleedout {
-            targetState = "Dead";
-            events[] = {QEGVAR(medical,Bleedout)};
```

Comment:
Believe this event may as well be removed entirely if we're making this change as its whole purpose was to kill a unit in cardiac arrest if they lost loads of blood (to the point of being non-recoverable where there's no point letting the timer run down).

Reply:
We can keep it, but behind a setting, I'd be fine with that. However in this current state, there is no way to get a somewhat similar behaviour to the old revive state, where you can always get units back up if they get treated within a certain timespan.

Reply:
To be honest I'm not even sure if it fulfils its original purpose, do we have much feedback on this front?

Reply:
People were asking about that in Slack, yes.

Reply:
My group and I are opposed to this change: being able to bleed to death while in cardiac arrest was a really positive point for the gameplay.

With this change, it's less interesting.

Might as well reduce the bleeding coefficient, if that's a problem?